TYPO3.TYPO3CR/Tests/Behavior/Features/Bootstrap/NodeOperationsTrait.php

```diff
@@ -866,6 +882,30 @@ public function iExpectToHaveUnpublishedNodesForTheFollowingContext($nodeCount,
     }
 
     /**
+     * @When /^I make the node visible$/
+     */
```

Comment:
maybe call this "I unhide the node"? Because, visibility depends on more than the "hidden" flag.